.github/workflows/build-and-test-pr.yaml

```
@@ -36,12 +36,21 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
-      - name: Upload coverage to Codecov
-        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+      - name: Check the coverage file existence
```

[Nits] How about this?

```suggestion
      - name: Check the existence of coverage files
```